### PR TITLE
Standardization

### LINKED ISSUE #5: Image at the bottom of readme file is broken.
screen shot 2017-06-01 at 11 36 06 pm

### PR DESCRIPTION
**Pull request by StuntsPT**
Added CONTRIBUTING.md, standardized README.md, improved "Standard JS" conformance, bumped package versions, removed spammy notifications and removed unimplemented tests. Fixes #5 .

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +42,7 @@
   },
   "scripts": {
     "preinstall": "sam/build.sh",
-    "test": "mocha --reporter spec",
+    "test": "standard && dependency-check .",
```
Comment (bmpvieira) [inline package.json]: Please keep mocha here after the other tests.

**Review by bmpvieira — CHANGES_REQUESTED** [review]
Just need to keep mocha in tests

**Comment by StuntsPT** [comment]
Added the `mocha` tests back. However, a new error shows up complaining there are no tests to perform...

**Comment by bmpvieira** [comment]
Sorry, I didn't see that tests were still missing. I'll merge your changes anyway and someone can add tests later.

**Review by bmpvieira — APPROVED** [review]

**Pull request merged**